homeassistant/components/tplink/light.py

```diff
@@ -60,6 +60,9 @@
 MAX_ATTEMPTS = 300
 SLEEP_TIME = 2
 
+FALLBACK_MIN_COLOR = 2500
+FALLBACK_MAX_COLOR = 6500
```

Comment:
2700 - 5000 would be accepted by more bulbs, see https://github.com/python-kasa/python-kasa/blob/master/kasa/smartbulb.py#L12 . If ignoring the US version of KL120 (which is not supported by pyhs100), then the range would be 2700 - 6500.